src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -662,6 +664,80 @@ static String getValidityInfoForSizeCappedPossiblyEmptyString(String fieldName,
         return "";
     }
 
+    /**
+     * Checks if Session Start Time is after 3 hours before now, before 90 days from now and at exact hour mark.
+     * @return Error string if {@code sessionStart} is more than 3 hours before now, after 90 days from now
+     *         or not at exact hour mark
+     *         Empty string if {@code sessionStart} is after 3 hours before now, before 90 days from now
+     *         and at exact hour mark
+     */
+    public static String getInvalidityInfoForStartTime(Instant startTime, String timeZone) {
+        boolean isExactHour = LocalDateTime.ofInstant(startTime, ZoneId.of(timeZone)).getMinute() == 0;
+        if (!isExactHour) {
+            return "The start time for this feedback session must be at exact hour mark.";
+        }
+        Instant threeHoursBeforeNow = ZonedDateTime
+                .ofInstant(TimeHelper.getInstantHoursOffsetFromNow(-3), ZoneId.of(Const.DEFAULT_TIME_ZONE))
+                .withZoneSameInstant(ZoneId.of(timeZone)).toInstant();
```

Comment:
Isn't this the same as `Instant.now().minus(Duration.ofHours(3))`?

Reply:
Unfortunately no. This is because `Instant.now()` is with reference to the default time zone, i.e. `UTC`, but the `Instant` to be compared with may not be in the `UTC` timezone.

Hm, maybe a more concise way is to modify `TimeHelper.getInstantHoursOffsetFromNow()` instead...

Reply:
> but the Instant to be compared with may not be in the UTC timezone

Not familiar with the context of this conversation, but have my doubts about this statement. Instant has no/should not have concept of different timezones, all values should be in UTC.

Reply:
Ah yes, I see. In the usual logic flow, when creating/updating feedback sessions, the `Instant` inputs are first adjusted to the UTC timezone prior to validations. However, now that the validations have changed, i.e. checking against absolute values, e.g. 3hrs before now, and there is no timezone adjustment when creating the sample sessions, I added the timezone adjustment in the validator method instead.

On top of that, since sample sessions, as suggested by @wkurniawan07, are not subject to the same restrictions, I shall get rid of the timezone adjustment in the validator method.